sqlglot/optimizer/qualify_tables.py

```diff
@@ -66,19 +53,9 @@ def qualify_tables(
                     if not source.args.get("catalog"):
                         source.set("catalog", exp.to_identifier(catalog))
```

Comment:
so it's not necessary anymore to do the select * on subqueries?

Reply:
Looks like it, but I need to check more examples to be sure + run the full optimizer on them to see what's up.

Reply:
I'll add it back again because it takes care of some issues in qualify columns I discovered.